packages/edge-demo-site/remix.config.js

```diff
@@ -8,11 +8,4 @@ module.exports = {
   // add your own custom config here if you want to.
   //
   // See https://remix.run/file-conventions/remix-config
-
```

Comment:
This used to be set to the root of the project because of how we had to handle builds pre-new monorepo support, but this is no longer required since the default configuration already handles it.

https://github.com/netlify/remix-compute/blob/main/packages/remix-edge-adapter/src/defaultRemixConfig.ts#L6